### PR TITLE
feat: Deprecation DEPPS9: LiveQuery `fields` option is renamed to `keys`

### LINKED ISSUE #8845: Remove deprecation DEPPS9: Rename LiveQuery `fields` option to `keys`
### New Feature / Enhancement Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).

### Feature / Enhancement Description

Remove deprecation DEPPS9: Rename LiveQuery `fields` option to `keys`

### References 

https://github.com/parse-community/parse-server/issues/8389

### PR DESCRIPTION
**Pull request by onurhanife**
## Pull Request

- Report security issues [confidentially](https://github.com/parse-community/parse-server/security/policy).
- Any contribution is under this [license](https://github.com/parse-community/parse-server/blob/alpha/LICENSE).
- Link this pull request to an [issue](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).

## Issue

Closes: #8845 

## Approach
In LiveQuery shema, deprecated fields options removed

Test results are same with alpha branch

## Tasks

- [ ] Add tests
- [ ] Add changes to documentation (guides, repository pages, code comments)
- [ ] Add [security check](https://github.com/parse-community/parse-server/blob/master/CONTRIBUTING.md#security-checks)
- [ ] Add new Parse Error codes to Parse JS SDK

### CODE REVIEW

File: spec/ParseLiveQueryServer.spec.js [hunk 1]
```diff
@@ -1131,7 +1131,7 @@ describe('ParseLiveQueryServer', function () {
     done();
   });
 
-  it('can deprecate fields', async () => {
+  xit('can deprecate fields', async () => {
```
Comment (mtrezza) [inline spec/ParseLiveQueryServer.spec.js]: Remove test if not needed anymore

**Review by mtrezza — CHANGES_REQUESTED** [review]

**Review by mtrezza — APPROVED** [review]
Looks good

**Review by mtrezza — APPROVED** [review]

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.0.0-alpha.16](https://github.com/parse-community/parse-server/releases/tag/7.0.0-alpha.16)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.0.0-beta.1](https://github.com/parse-community/parse-server/releases/tag/7.0.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.0.0](https://github.com/parse-community/parse-server/releases/tag/7.0.0)

**Pull request merged**